packages/framework/src/Resources/config/services.yml

```diff
@@ -465,10 +465,6 @@ services:
         arguments:
             - '@logger'
 
-    Shopsys\FrameworkBundle\Model\Mail\MailTemplateFacade:
```

Comment:
You shouldn't remove a service definition because of backward-compatibility, even if it's not currently specified in our BC promise...

Reply:
I've created https://github.com/shopsys/shopsys/issues/1165 to address the incomplete BC promise.

Reply:
```suggestion
    Shopsys\FrameworkBundle\Model\Mail\MailTemplateFacade: ~ # @deprecated
```

Reply:
Even when the facade is defined automatically with the suffix?

Reply:
I haven't thought of that, good point :+1:

It got me thinking whether we have more such redundancies, so I opened https://github.com/shopsys/shopsys/pull/1166